Firmware/Marlin_main.cpp

```diff
@@ -2825,7 +2825,7 @@ static void gcode_G80()
         nMeasPoints = 3;
     }
 
-    uint8_t nProbeRetry = code_seen('R') ? code_value_uint8() : eeprom_read_byte((uint8_t*)EEPROM_MBL_PROBE_NR);
+    uint8_t nProbeRetry = code_seen('C') ? code_value_uint8() : eeprom_read_byte((uint8_t*)EEPROM_MBL_PROBE_NR);
```

Comment:
Maybe also change `nProbeRetry` to `nProbeCount` as well

Reply:
Maybe nProbeRetryCount would be better? It will still do 1 Probe if set to 0. (2 if set to 1 etc.)